src/components/HeaderBar.vue

```diff
@@ -91,30 +106,67 @@ export default {
 			this.newTaskName = ''
 		},
 
-		addTask() {
-			const task = { summary: this.newTaskName }
-
+		addTaskWithName(task) {
 			// If the task is created in the calendar view,
 			// we set the current calendar
 			if (this.$route.params.calendarId) {
 				task.calendar = this.calendar
 			}
 
+			return this.createTask({
+				...task,
+				...this.getAdditionalTaskProperties()
```

Comment:
This is a computed prop, I don't think you need to call it with `()`, no?

Reply:
This is actually a method, see https://github.com/nextcloud/tasks/pull/2273/files#diff-6648f3fa5be4b95d4036efb24c229f414ab0f55274f6ecc7c150708d90d674b5R121